appveyor.yml

```diff
@@ -24,8 +24,10 @@ install:
   - choco install -y googlechrome --ignore-checksums
   # Install the latest stable version of Node
   - ps: Install-Product node $env:nodejs_version
-  - npm install npm -g
-  - npm ci
+  - npm install
+
+# TODO: Switch to using npm ci instead of install
+# once Node LTS ships with npm 5.7 (with ci)
```

Comment:
node 8.12 (lts) comes with npm 6.4

Reply:
Thanks for the notice! It wasn't in 8.11. Will check if Appveyor provides 8.12 already and update accordingly!